### PR TITLE
Use hash ref in query_isotovideo and add a test for it.

### PR DESCRIPTION
**Pull request by foursixnine**
Supersede  #852

### CODE REVIEW

File: t/data/tests/tests/shutdown.pm [hunk 1]
```diff
@@ -18,6 +18,7 @@ use base 'basetest';
 use testapi;
 
 sub run {
+    wait_idle 1;
```
Comment (okurz) [inline t/data/tests/tests/shutdown.pm]: IMHO a waste of time. wait_idle should be tested in t/03-testapi.t with mocked sleep

**Comment by okurz** [comment]
* Please don't push merge commits, rebase
* Please don't add a dot at EOL of the git commit message subject line

**Comment by foursixnine** [comment]
@okurz Yeah, happens when the gh editor is used :( That thing is evil for this kind of things...

**Comment by coolo** [comment]
no travis?

**Comment by foursixnine** [comment]
@coolo Travis seems to be drunk again :(

**Comment by foursixnine** [comment]
Now we're talking!

File: t/data/tests/tests/shutdown.pm (hunk 1 above)
Comment (okurz) [reply t/data/tests/tests/shutdown.pm]: So you don't want to comment on this?

**Pull request merged**